src/extension.c

```diff
@@ -15,12 +15,51 @@
 #include "query.h"
 #include <err.h>
 
+#define MAX_NAME_LEN 1024
```

Comment:
Consider making it 1025 bytes long, so it would be easier to document that a scorer max name length is 1024 ascii characters